packages/salesforcedx-vscode-aura/src/commands/quickpick/quickpick.ts

```diff
@@ -0,0 +1,55 @@
+import { paramCase } from 'change-case';
+import opn = require('opn');
+import { Uri, window, workspace } from 'vscode';
+import { LanguageClient } from 'vscode-languageclient';
+import { LwcNode } from '../../views/lwc-node';
+import { TagItem } from './tag-item';
+
+export function createQuickOpenCommand(client: LanguageClient) {
+    return (arg: any) => {
+        if (arg instanceof LwcNode) {
+            const node = arg as LwcNode;
+            const tag: string = node.label;
+            opn(`https://developer.salesforce.com/docs/component-library/bundle/${tag}`);
```

Comment:
We normally have urls on a `constants.ts` per module. If it's part of a message or it needs to be dynamically modified then we also use `i18n.ts` under each package's `messages` folder. I think it's up to you which of these options we go with for this.

Reply:
I created a constants file